### PR TITLE
{mpi}[intel/2015b] mpi4py 1.3.1 & 2.0.0 w/ Python 2.7.10 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/m/mpi4py/mpi4py-1.3.1-intel-2015b-Python-2.7.10.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonPackage'
+
+name = 'mpi4py'
+version = '1.3.1'
+
+homepage = 'https://bitbucket.org/mpi4py/mpi4py'
+description = """MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for
+ the Python programming language, allowing any Python program to exploit multiple processors."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/mpi4py/mpi4py-1.3.1-intel-2015b-Python-2.7.10.eb]: no need for `usempi`?

**Comment by wpoely86** [comment]
lgtm

File: easybuild/easyconfigs/m/mpi4py/mpi4py-1.3.1-intel-2015b-Python-2.7.10.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/m/mpi4py/mpi4py-1.3.1-intel-2015b-Python-2.7.10.eb]: nope, `$MPICC` is used below, usempi means defining `$CC=$MPICC`, which is not needed here

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/df8a345fdf73494953c4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d1ab85cf3b2bee50081e for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**